org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/ClientPreferences.java

```diff
@@ -219,6 +219,14 @@ public boolean isExtractMethodInferSelectionSupport() {
 		return false;
 	}
 
+	public boolean isExtractFieldInferSelectionSupport() {
```

Comment:
isExtractFieldInferSelectionSupported

Reply:
Fixed. Will change `isExtractMethodInferSelectionSupport` to `isExtractMethodInferSelectionSupported` as well.